docker-for-mac/apple-m1.md

```diff
@@ -12,17 +12,21 @@ Welcome to Docker Desktop for Apple Silicon.
 >
 > We encourage you to try the release candidate and report any issues in the [Docker Desktop for Mac GitHub](https://github.com/docker/for-mac) repository.
 
-## Docker Desktop RC 1
+## Docker Desktop RC 2
 
-2021-03-18
+2021-03-25
 
 Click on the following link to download the latest release candidate of Docker Desktop for Apple Silicon.
 
-> [Download](https://desktop.docker.com/mac/stable/arm64/62029/Docker.dmg)
+> [Download](https://desktop.docker.com/mac/stable/arm64/62345/Docker.dmg)
+
+This build allows users to switch between `qemu`-based and `virtualization.framework`-based virtual machines. The default is `qemu`, but it can be switched back to `virtualization.framework` in the experimental tab of the Preferences.
+
+This is still a tech preview, and we are not committing to include either of these specific back ends for the final release. Which we release eventually will depend on user feedback. We hope that the new `qemu` back end will avoid some of the known issues listed below, and we would welcome confirmation of this. However, it may introduce other issues that we have not discovered in in-house testing, so we would also welcome reports of problems with it, especially areas where it is inferior to the `virtualization.framework`.
```

Comment:
```suggestion
Apple Silicon for Docker Desktop is still a tech preview. We are not committing to include either of these specific backends for the final release. The backend that we choose to release eventually depends on user feedback. We hope that the new `qemu` backend will avoid some of the known issues listed below. We welcome your confirmation on this. However, it may introduce other issues that we have not discovered in in-house testing, so we would also welcome any reports of problems with it, especially areas where it is inferior to the `virtualization.framework`.
```